profiles/mukul314.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/76463001/190381823-6023c142-cf15-45b7-9117-a19fc9deb098.png",
```

Comment:
What do you think of this image?

![image](https://user-images.githubusercontent.com/51391473/190681258-98264c81-a1f6-454c-999a-f2f089fd78d0.png)

```suggestion
  "image":"https://user-images.githubusercontent.com/51391473/190681258-98264c81-a1f6-454c-999a-f2f089fd78d0.png",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant